RecoJets/JetPlusTracks/python/PATJetPlusTrackCorrections_cff.py

```diff
@@ -0,0 +1,105 @@
+import FWCore.ParameterSet.Config as cms
+# ---------- Add assigned jet-track association
+
+from RecoJets.JetAssociationProducers.trackExtrapolator_cfi import *
+trackExtrapolatorJPTPAT = trackExtrapolator.clone(
+                      trackSrc = "trackFromPackedCandidate",
+                      trackQuality = 'highPurity'
+)
+
+from RecoJets.JetAssociationProducers.ak4JTA_cff import *
+ak4JetTracksAssociatorAtVertexJPTPAT = ak4JetTracksAssociatorAtVertex.clone(
+                                       useAssigned = True,
+                                       pvSrc = "offlineSlimmedPrimaryVertices",
+                                       jets = "slimmedCaloJets",
+                                       tracks = "trackFromPackedCandidate"
+)
+ak4JetTracksAssociatorAtCaloFaceJPTPAT = ak4JetTracksAssociatorAtCaloFace.clone(
+                                         jets = "slimmedCaloJets",
+                                         tracks = "trackFromPackedCandidate",
+                                         extrapolations = "trackExtrapolatorJPTPAT"
+)
+ak4JetExtenderJPTPAT = ak4JetExtender.clone(
+                       jets = "slimmedCaloJets",
+                       jet2TracksAtCALO = "ak4JetTracksAssociatorAtCaloFaceJPTPAT",
+                       jet2TracksAtVX = "ak4JetTracksAssociatorAtVertexJPTPAT"
+)
+
+# ---------- Supported Modules
+
+from CommonTools.RecoAlgos.trackFromPackedCandidateProducer_cfi import *
+trackFromPackedCandidate = trackFromPackedCandidateProducer.clone(PFCandidates = 'packedPFCandidates')
+
+
+from RecoJets.JetPlusTracks.jetPlusTrackAddonSeedProducer_cfi import *
+JetPlusTrackAddonSeedRecoPAT = jetPlusTrackAddonSeedProducer.clone(
```

Comment:
I'm just realizing now that this should be named ```JetPlusTrackAddonSeedPAT```, because Reco is a part of the name in ```JetPlusTrackAddonSeedReco```, which is meant to refer to a setup running in Reco.